web/packages/teleterm/src/ui/utils/assertUnreachable.ts

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+/**
+ * @deprecated Import assertUnreachable from `shared/utils/error` instead.
+ */
 export function assertUnreachable(x: never): never {
   throw new Error(`Unhandled case: ${x}`);
 }
```

Comment:
Thanks for this. 
Small improvement: we can just re-export the function from `shared` here, instead of declaring it again.